scalableminds/webknossos
# Disable UI starting dataset creating jobs for unprivileged users

This PR disables the UI elements allowing the creation of a new dataset as an output of a job for normal unprivileged users.

### URL of deployed dev instance (used for testing):
- https://onlyallowprivilegdeuserdatasetcreationjobs.webknossos.xyz/
### Steps to test:
- On the dev instance log in as the sample2@scm.io user and try to access UI allowing you to start a job creating a dataset:
- `<host-ulr-part>/datasets/upload` should now show a 403 error page
  - For this, an improved icon might be preferable that follows wks drawing style
- In tracing view:
  - The "materialize merger mode annotation" button should not be rendered
  - The AI Analysis Button should not be rendered
  - The option "Merge this volume annotation with its fallback layer" of a segmentation layer with fallback data (e.g. l4_sample) should not be rendered

### Open question:
- Is there a UI element I missed while scanning the codebase?

### TODOs:
- [x] Confirm whether keeping the 403 page is ok
  - [x] Get a better image for the forbidden page

### Issues:
- fixes [#7734](https://github.com/scalableminds/webknossos/issues/7734)

------
(Please delete unneeded items, merge only when none are left open)
